diff --git a/pom.xml b/pom.xml
index 0c72eea..8b68c3c 100644
--- a/pom.xml
+++ b/pom.xml
@@ -330,7 +330,7 @@
             <dependency>
                 <groupId>jakarta.enterprise</groupId>
                 <artifactId>jakarta.enterprise.cdi-api</artifactId>
-                <version>2.0.SP1</version>
+                <version>4.0.1</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
diff --git a/tests-configuration-batch/src/test/java/org/jboss/hal/testsuite/test/configuration/batch/Batchlet.java b/tests-configuration-batch/src/test/java/org/jboss/hal/testsuite/test/configuration/batch/Batchlet.java
index dfb4f26..a2b6a87 100644
--- a/tests-configuration-batch/src/test/java/org/jboss/hal/testsuite/test/configuration/batch/Batchlet.java
+++ b/tests-configuration-batch/src/test/java/org/jboss/hal/testsuite/test/configuration/batch/Batchlet.java
@@ -4,8 +4,8 @@ import java.util.concurrent.TimeUnit;
 
 import jakarta.batch.api.AbstractBatchlet;
 import jakarta.batch.api.BatchProperty;
-import jakarta.inject.Inject;
-import jakarta.inject.Named;
+import javax.inject.Inject;
+import javax.inject.Named;
 
 import org.jboss.logging.Logger;
 
diff --git a/tests-configuration-undertow/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/Counter.java b/tests-configuration-undertow/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/Counter.java
index c7a00f4..82990b2 100644
--- a/tests-configuration-undertow/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/Counter.java
+++ b/tests-configuration-undertow/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/Counter.java
@@ -1,7 +1,7 @@
 package org.jboss.hal.testsuite.test.configuration.undertow;
 
-import jakarta.inject.Named;
-import jakarta.inject.Singleton;
+import javax.inject.Named;
+import javax.inject.Singleton;
 
 @Named
 @Singleton
